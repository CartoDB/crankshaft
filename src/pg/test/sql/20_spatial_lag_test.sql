\pset format unaligned
\set ECHO all
\i test/fixtures/spatial_lag_file.sql

-- Spatial Lag test

SELECT m.rowid, m.spatial_lag
  FROM spatial_lag_file
  JOIN cdb_crankshaft.CDB_SpatialLag('SELECT * FROM spatial_lag_file', 'value', 'knn',5, 'the_geom','cartodb_id') m
    ON spatial_lag_file.cartodb_id = m.rowid
  ORDER BY spatial_lag_file.cartodb_id;
