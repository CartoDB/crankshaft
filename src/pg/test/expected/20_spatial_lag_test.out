\pset format unaligned
\set ECHO all
\i test/fixtures/spatial_lag_file.sql
SET client_min_messages TO WARNING;
\set ECHO none
rowid|spatial_lag
26|1.5558245520965668
36|0.86458398182170748
47|0.95270659529918711
48|1.2321749304579666
52|0.7067108549416905
63|1.7648393915166776
70|8.5933781209139841
74|8.585129430302981
81|1.2762005195217829
96|8.5862165805895536
97|4.4808366319910027
112|11.952508899443769
113|9.8633331700241413
116|11.950331957611654
117|4.6317289221482971
119|1.5003998060451806
125|4.623201011124598
140|0.81320866139607006
143|1.1652798506659032
145|1.0029589151893603
146|0.73142355952732685
148|0.89698376389437184
151|0.89265718517960235
153|4.4672874115845964
154|0.79363641536354224
160|4.4995909659653517
174|0.99031984840512266
201|0.9850340860853366
203|0.95995789366679918
205|0.44649384203794357
210|0.84287567979703126
216|0.45638121464530523
221|0.83657469267602325
222|0.32468328149670383
226|0.96856924566947733
234|0.98029560059066778
236|0.36577884387846982
239|0.46532563118851461
265|0.55308241153547621
274|0.37779583501369646
(40 rows)
